src/app/modules/auth-private/sign-up/sign-up.component.ts

```diff
@@ -155,19 +155,22 @@ export class SignUpComponent implements OnInit {
     ).subscribe((token) => {
       if (token) {
         this.tokenService.storeToken(token);
-        this.getCurrentUser();
```

Comment:
@Harshwardhan18 Can we do here something like this:
If the person has not filled up organization and designation, only then we show the next step where he'll be able to add it. In case if it is already present, we do not need to go to next step

Reply:
@Manzurkds How we'll now he hasn't filled the data? The signup object won't have the data anyways before it's initialized

Reply:
@Harshwardhan18 There was a possible case of the data being added from the admin panel you said right? In that case, it could be present

Reply:
@Manzurkds No case bro sedhu confirmed it.